app/src/main/res/layout-land/onboarding_slide.xml

```diff
@@ -37,6 +37,7 @@
 
       <TextView
         android:id="@+id/slide_title_text_view"
+        style="@style/Heading1"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
```

Comment:
```
android:layout_width="0dp"
android:layout_height="wrap_content"
```

The above lines can be removed and you can introduce `horizontalBias=0.0`.

Considering that the Heading1 already have these attributes.

Reply:
@rt4914 The Heading1 style has basically two attributes, the  fontFamily and the textSize, can you take another look at it.
  <style name="Heading1" parent="TextView.Common1">
    <item name="android:fontFamily">sans-serif</item>
    <item name="android:textSize">24sp</item>
    <item name="fontFamily">sans-serif</item>
  </style>

Reply:
I have removed the redundant `android:layout_height="wrap_content"` line.